hack/dev-env.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+RDR_NAME_PREFIX=${RDR_NAME_PREFIX:-rdr}
+script_dir="$(cd "$(dirname "$0")" && pwd)"
+old_cwd=$(pwd)
+
+function finish {
+    cd "$old_cwd"
+}
+trap finish EXIT
```

Comment:
We don't need this since changing directory in a script happens in the new shell and does not affect the original shell.

Reply:
Ack, will remove the trap.